pkg/render/compliance.go

```diff
@@ -675,10 +675,6 @@ func complianceAnnotations(c *complianceComponent) map[string]string {
 		ManagerTLSHashAnnotation:          AnnotationHash(c.managerSecret),
```

Comment:
Doesn't this line mean the annotation is still set?

Reply:
Yeah, but it does not seem to matter. I removed the method.

Reply:
Yes, for compliance we still have to set an annotation. The problematic one is the one for the secret.